lib-compose/src/main/java/com/what3words/components/compose/maps/W3WMapManager.kt

```diff
@@ -203,11 +205,13 @@ class W3WMapManager(
     //endregion
 
     // region Camera control
-    fun orientCamera() {
+    @UiThread
+    suspend fun orientCamera() {
         mapState.value.cameraState?.orientCamera()
     }
 
-    fun moveToPosition(
+    @UiThread
```

Comment:
The @UiThread annotation in Android is a contract or indicator, not a guarantee and it does not enforce it at runtime or switch to the main thread automatically. To ensure thread safety and compliance with coroutine rules, please modify the function to explicitly run on the Dispatchers.Main dispatcher